README.md

```diff
@@ -55,12 +55,6 @@ Note: `python` and `vips` are dependencies via `sharp`.
 bun install
```

Comment:
removing instructions related to pre-building/watching web-components + cleaning up